aws-core/src/main/java/software/amazon/awssdk/awscore/exception/AwsServiceException.java

```diff
@@ -51,9 +52,11 @@ public boolean isClockSkewException() {
     }
 
     @Override
-    public boolean additionalThrottlingCondition() {
-        return Optional.ofNullable(errorCode())
+    public boolean isThrottlingException() {
+        return super.isThrottlingException() ||
```

Comment:
Not against it, but I created the hook method `additionalThrottlingCondition` because I feel calling super is a bit code smell as you have to always remember to call super for every sub classes of SdkServiceException. https://www.martinfowler.com/bliki/CallSuper.html

Reply:
I typically agree as you may forget to do it but I changed it mainly because the hook method had no clear good name, additionalThrottlingCondition didn't sound quite right to me. And also it forces a certain implementation on subclasses which may or may not be appropriate so I'd rather have them opt into the super behavior.